server/views/pages/prisons/addSessionTemplate.njk

```diff
@@ -0,0 +1,254 @@
+          name: "dayOfWeek",
+          label: {
+            text: "Day of the week",
+            classes: "govuk-label--m"
+
+          },
+          items: [
+            {
+              value: "choose",
+              text: "Choose day of the week",
+              selected: true
+            },
+            {
+              value: "MONDAY",
+              text: "Monday"
+            },
+            {
+              value: "TUESDAY",
+              text: "Tuesday"
+            },
+            {
+              value: "WEDNESDAY",
+              text: "Wednesday"
+            },
+            {
+              value: "THURSDAY",
+              text: "Thursday"
+            },
+            {
+              value: "FRIDAY",
+              text: "Friday"
+            },
+            {
+              value: "SATURDAY",
+              text: "Saturday"
+            },
+            {
+              value: "SUNDAY",
+              text: "Sunday"
+            }
+          ]
+
+        }) }}
+
+        {{ govukInput({
+          label: {
+              text: "Visit start time",
+              classes: "govuk-label--m"
+          },
+          hint: {
+              text: "For example, 13:00"
+          },
+          classes: "govuk-!-width-one-third",
+          id: "startTime",
+          name: "startTime",
+          autocomplete: "off",
+          errorMessage: errors | findError('startTime')
+        }) }}
+
+        {{ govukInput({
+          label: {
+              text: "Visit end time",
+              classes: "govuk-label--m"
+          },
+          hint: {
+              text: "For example, 13:00"
+          },
+          classes: "govuk-!-width-one-third",
+          id: "endTime",
+          name: "endTime",
+          autocomplete: "off",
+          errorMessage: errors | findError('endTime')
+        }) }}
+
+        {{ govukRadios({
+          name: "biWeekly",
+          fieldset: {
+            legend: {
+              text: "Bi-weekly?",
+              classes: "govuk-fieldset__legend--m"
+            }
+          },
+          items: [
+            {
+              value: "true",
+              text: "True"
+            },
+            {
+              value: "false",
+              text: "False"
+            }
+          ]
+        }) }}
+
+        {{ govukDateInput({
+          id: "validFromDate",
+          namePrefix: "validFromDate",
+          fieldset: {
+            legend: {
+              text: "Template start date",
+              classes: "govuk-fieldset__legend--m"
+            }
+          },
+          hint: {
+            text: "For example, 27 3 2023"
+          }
+        }) }}
+   
+        {{ govukCheckboxes({
+          name: "hasEndDate",
+          fieldset: {
+            legend: {
+              text: "Does this session have an end date?",
+              classes: "govuk-fieldset__legend--m"
+            }
+          },
+          items: [
+            {
+              value: "hasEndDate",
```

Comment:
Might just want this to be "true" or "yes"? i.e. the checkbox field name will be `hasEndDate` and it's value could be, say, `yes`. Doesn't really matter though - see how it works out...